readme.md

```diff
@@ -1,115 +1,96 @@
-
-In addition, the script contains functionality for changing values by column and row using a yaml configuration. An example of this yaml configuration is located in `python/configurations/` as 
-[`row_by_row_modification.yaml.`](https://github.com/cfpb/hmda-test-files/blob/master/python/configurations/row_by_row_modification.yaml) The script can contain a path to a yaml configuration to initiate this functionality. 
-
-## Getting Started: Generating Clean Files, Edit Test Files, and Large Files. 
-The function method `create_files(kind)` outputs a clean test file or a set of edit test files, with an input indicating the kinds of file to be created, `"clean_file"` or `"error_files"`. The `kind` keyword in the function is inputted as a string. 
-
-A suggested workflow for creating test files:
+## Generating Clean Files
+Clean files will pass the HMDA edits (business rules for data submission). These files are used as the base for generating files that will fail edits. Running the following scripts will create the edits_files directory and a data file that will pass the HMDA edit checks. The file will have a number of rows set in a YAML clean file configuration for each directory. Other variables, such as data ranges can also be set in the configuration files.
 
-1) *Instantiate FileGenerator*: Instantiate `FileGenerator()` to a python object in a script in the `python/` directory. Such as `file = FileGenerator()`.
+For 2019:
+1. Run `cd 2019/python` to navigate to the python directory. 
+2. Adjust the [2019 Clean File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/clean_file_config.yaml) for bank name, LEI, Tax ID and number of rows. 
+3. Run `python3 generate_2019_clean_files.py`
+4. The clean test file should appear in a new edits_files directory under `2019/edits_files/clean_files/{Bank Name}/` with the filename `clean_file_{Number or Rows}_{Bank Name}.txt`. 
 
-2) *Adjust Configuration Files*: Make modifications to `test_filepaths.yaml`, `geography_data.yaml`, and `clean_file_config.yaml` for the kind of clean file to be produced. 
+Note: the filepath and filename for clean files can be adjusted in the [2019 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml) YAML configuration file under `clean_filepath` and `clean_filename` keys. 
 
-3) *Create a Clean File*: Run `file.create_files(kind='clean_file')` in the script to produce a clean file saved in a directory specified in `test_filepaths.yaml`. 
+For 2018:
+1. Run `cd 2018/python` to navigate to the python directory. 
+2. Adjust the [2018 Clean File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/clean_file_config.yaml) for bank name, LEI, Tax ID and number of rows. 
```

Comment:
Same comment as above, the user should not have to adjust defaults to run the code. This is an optional feature. Please make the same change as above.